cart.php

```diff
@@ -94,12 +94,14 @@
         INNER JOIN item i on c.item_id = i.id WHERE user_id =' . $user_id . ' ';
 
         $stmt = $db->prepare($sql);
-
         $stmt->execute();
+
         $query->CustomSQL('DELETE FROM cart WHERE user_id = :user_id', $params);
 
         $_SESSION['message'] = 'Thanks for your purchase, an email will be sent to you shortly with your order receipt';
 
+        $modifyCart->emailItems();
```

Comment:
Good instinct here to delegate the actual email-sending to a class function rather than shoe-horning it within `cart.php`